src/index.js

```diff
@@ -107,6 +107,7 @@ const initial_state = {
   inventory_sequence_number: 0,
   inventory_cursor: null,
   inventory_cursor_index: 0,
+  inventory_held_slot_index: 0, // goes from 0 to 8
```

Comment:
Don't want to describe that with a type :smile:  ?
```suggestion
  /** @type {0|1|2|3|4|5|6|7|8} */ 
  inventory_held_slot_index: 0,
```

Reply:
I think we should remove the `inventory_` prefix here

Reply:
> Don't want to describe that with a type 😄 ?

![image](https://user-images.githubusercontent.com/11330271/130368445-365e3c68-545f-4568-a2b2-863c504ae99f.png)